packages/server/src/auth/api/member.ts

```diff
@@ -1,24 +1,34 @@
-import * as Prisma from '@prisma/client'
+import { Member } from '@prisma/client'
 import { intArg, mutationField, nonNull, objectType, queryField, stringArg } from 'nexus'
-import { Member } from 'nexus-prisma'
+import { Member as NexusMember } from 'nexus-prisma'
+import { error } from 'npmlog'
+import * as Yup from 'yup'
 
-import { authMemberId, verifyEmailToken } from '@/auth/model/token'
+import { verifyEmailToken } from '@/auth/model/token'
 import { Context } from '@/common/api'
 
+import { sendVerificationEmail } from '../model/emailVerification'
+
 export const MemberFields = objectType({
-  name: Member.$name,
-  description: Member.$description,
+  name: NexusMember.$name,
+  description: NexusMember.$description,
   definition(t) {
-    t.field(Member.id)
-    t.field(Member.name)
-    t.field(Member.email)
+    t.field(NexusMember.id)
+    t.field(NexusMember.name)
+    t.field(NexusMember.email)
   },
 })
 
 export const MemberQuery = queryField('member', {
-  type: Member.$name,
+  type: NexusMember.$name,
+
+  resolve: (_, __, { member }: Context) => {
+    if (!member) {
+      throw new Error('Unauthorized')
+    }
```

Comment:
I think I disagree. It's not really public since it will only return any value if you are logged in. WDYT about renaming query to `me`? I think that's a common pattern to look up the user you are logged in as.

Reply:
Cool, renamed the query (also in docs)